app/lib/report_helper.rb

```diff
@@ -438,6 +438,9 @@ def self.fake_genus!(tax_info)
     # This is a workaround for a bug we are fixing soon... but leaving
     # in the code lest that bug recurs.  A warning will be logged.
     #
+    # Two years ago, the above comment was written. It is not clear whether the
```

Comment:
do you want to date these comments? 😅

Reply:
Should probably generate an alert for this... Make the bug more visible.

Reply:
I thought git blame was enough for dating the comments. 

As for the alert, I think it is better to ask @boris-dimitrov or whoever would know and try to fix to root cause. There was a log warning already but we ignored it.

Reply:
Exactly because it was ignored, we have to raise its visibility to an error+alert. Until we fix it. It was ignore because it was a warning...

Reply:
OK, maybe we need to clarify this logic before raising a more drastic alert...

Reply:
@jshoe said it was related to an old "hit calling" feature which affected samples for a few weeks. So it should still work correctly as a workaround for the report page. No need to raise an alert.